### PR TITLE
feat(permissions): allow run permission to take values

### LINKED ISSUE #9925: Run permission can be scoped/limited
# Problem

Hi! 
In the Deno project i use Python scripts. I do this with Deno.run variable:

```js
Deno.run({
        cmd: ['python', 'python/example.py']
})
```

This is an incredibly useful feature, but for this I need to specify a permission flag  `--allow-run`.  
`--allow-run` flag open access to run any processes and it is not secure!

# Idea

Is it possible to add a more detailed permission in the future, like `--allow-read` or `--allow-write`? 
Example:

```sh
deno run --allow-run=./python/  app.ts
```

or

```sh
deno run --allow-scripts app.ts
```

or

```sh
deno run --allow-scripts=example.py,example.rs app.js
```

Thanks!

### PR DESCRIPTION
**Pull request by crowlKats**
Closes #9925

### CODE REVIEW

File: cli/flags.rs [hunk 1]
```diff
@@ -1808,12 +1819,15 @@ fn permission_args_parse(flags: &mut Flags, matches: &clap::ArgMatches) {
     debug!("net allowlist: {:#?}", &flags.allow_net);
   }
 
+  if let Some(run_wl) = matches.values_of("allow-run") {
+    let run_allowlist: Vec<String> = run_wl.map(ToString::to_string).collect();
+    flags.allow_run = Some(run_allowlist);
+    debug!("env allowlist: {:#?}", &flags.allow_run);
```
Comment (bartlomieju) [inline cli/flags.rs:1825]: Remove?

File: cli/flags.rs [hunk 2]
```diff
@@ -1398,6 +1405,10 @@ fn permission_args<'a, 'b>(app: App<'a, 'b>) -> App<'a, 'b> {
     .arg(
       Arg::with_name("allow-run")
         .long("allow-run")
+        .min_values(0)
+        .takes_value(true)
+        .use_delimiter(true)
+        .require_equals(true)
```
Comment (bartlomieju) [inline cli/flags.rs:1411]: I think some kind of validator is needed here, similar to `--allow-env` PR

Comment (crowlKats) [reply cli/flags.rs:1411]: sure, but how would we validate passed commands? check if they actually exist?

**Comment by bartlomieju** [comment]
@crowlKats this PR looks good to me, let's land it for 1.9. Please rebase

**Review by bartlomieju — COMMENTED** [review]
@crowlKats before landing can you add an integration test to `cli/tests/integration_tests.rs` that uses an allow list? Something like `_045_proxy` that runs Deno executable.

**Review by bartlomieju — COMMENTED** [review]
LGTM, great feature @crowlKats!

**Review by bartlomieju — APPROVED** [review]

**Pull request merged**